### PR TITLE
Splash page sponsers section style improvement

### PR DESCRIPTION
**Pull request by sonalkr132**
supersedes #951 

- [x] make logo of a given sponsorship level of same size
- [x] pop up model for sponsor desc
- [x] test and documentation

### CODE REVIEW

**Comment by sonalkr132** [comment]
so far:
screenshot from 2016-04-20 12-06-40

**Comment by sonalkr132** [comment]
pop up with description:
screenshot from 2016-04-20 13-33-03

without description:
screenshot from 2016-04-20 13-34-14

**Comment by hennevogel** [comment]
Again :-1: on showing the sponsor levels to the outside world

**Comment by sonalkr132** [comment]
Which one looks better?
Left-align:
screenshot from 2016-04-20 22-13-57
or center:
screenshot from 2016-04-20 22-15-15

Note: rest of page is center aligned.

**Comment by hennevogel** [comment]
I like the pyramid. It conveys that the top sponsors have done more.

File: INSTALL.md [hunk 1]
```diff
@@ -63,6 +63,15 @@ We recommend to run OSEM in production with [mod_passenger](https://www.phusionp
 and the [apache web-server](https://www.apache.org/). There are tons of guides on how to deploy rails apps on various
 base operating systems. Check Google ;-)
 
+#### ImageMagick
+We use imagemagic for image manipulation of sponsor logo. You can get it from [direct install](http://software.opensuse.org/package/ImageMagick) page of the package or else check out [Download page](http://www.imagemagick.org/script/binary-releases.php) of ImageMagick.
+
+If you are upgrading your osem instance and would like to resize the exisiting logos, you would need to `reprocess!` the images:
+```
+$ rails c
```
Comment (hennevogel) [inline INSTALL.md]: Please create a rake job for this

Comment (sonalkr132) [inline INSTALL.md]: Good idea! It's done now.

**Comment by sonalkr132** [comment]
> I like the pyramid. It conveys that the top sponsors have done more.

This PR has implementation of pyramid (or center alignment)

**Pull request merged**